tests/orchestrator/test_call_http.py

```diff
@@ -174,3 +175,36 @@ def test_post_completed_state():
     # assert_valid_schema(result)
     assert_orchestration_state_equals(expected, result)
     validate_result_http_request(result)
+
+@pytest.mark.parametrize("content, expected_content, is_raw_str", [
```

Comment:
Let's please add a test that checks the method throws if `is_raw_str` is `True` and `content` is not an `str`

Reply:
Test added